src/requires.jl

```diff
@@ -15,19 +15,24 @@ function __init__()
     end)
 
     @require Colors="5ae59095-9a9b-59fe-a467-6f913c188581" begin
+        function _rgb_for_printing(c::Colors.Colorant)
+            rgb = convert(Colors.RGB{Float64}, c)
+            round.((Colors.red(rgb), Colors.green(rgb), Colors.blue(rgb)); digits = 4) # somewhat arbitrary choice of accuracy
+        end
+
         function PGFPlotsX.print_opt(io::IO, c::Colors.Colorant)
-            rgb = convert(Colors.RGB, c)
-            rgb_64 = convert.(Float64, (rgb.r, rgb.g, rgb.b))
+            rgb_64 = _rgb_for_printing(c)
             print(io, "rgb,1:",
                   "red,"  , rgb_64[1], ";",
                   "green,", rgb_64[2], ";",
                   "blue," , rgb_64[3])
         end
 
+        PGFPlotsX.print_tex(io::IO, c::Colors.Colorant) = PGFPlotsX.print_opt(io, c)
```

Comment:
Note that `print_opt` [falls back to](https://github.com/KristofferC/PGFPlotsX.jl/blob/23fe9ea28fcaa30a64eb2f511725d34e5603c3c6/src/options.jl#L161) `print_tex`, so you only need to define the latter where no special case is needed for printing in the context of options.

Reply:
Thanks, I didn't see where print_opt = print_tex